### DISCUSSION TITLE
How does phone home work

### DISCUSSION THREAD

**Discussion by paulmortlock** [sam-ludlow/mame-ao#48] [General]
Can we see who has played what and when and who they are?

**Comment by sam-ludlow** [comment]
Hello @paulmortlock 

Thanks for the question, yeah plan is to report on all this stuff. At the moment just collecting the payloads.

I will be adding some stats to start with here https://mame.spludlow.co.uk/MAME-AO.aspx

Currently I can see what is being played and for how long. Probably add more stuff like MAME version etc...

I'm just dusting off some old whois code to convert IP addresses to country.

Here are some raw numbers if you are interested.

Date	RequestCount
2024-08-17	79
2024-08-18	99
2024-08-19	239
2024-08-20	215
2024-08-21	175
2024-08-22	154
2024-08-23	242
2024-08-24	121
2024-08-25	279
2024-08-26	120
2024-08-27	190
2024-08-28	194
2024-08-29	244
2024-08-30	127
2024-08-31	18

Will let you know here.

Cheers
Sam


```
{
  "message_id": "cd3f0ce3-9415-4022-893f-b07e1afb2c9d",
  "start_time": "2024-08-31T10:19:15.8406302+10:00",
  "ready_time": "2024-08-31T10:19:36.3203546+10:00",
  "end_time": "2024-08-31T10:19:52.8055269+10:00",
  "line": "contra",
  "mame_exit_code": 0,
  "mame_output": "Average speed: 100.00% (10 seconds)\r\n",
  "mame_error": "Warning: layout view 'Upright_Artwork' contains deprecated bezel element\r\nWarning: layout view 'Upright_Artwork' contains deprecated backdrop element\r\nWarning: layout view 'XBLA_Widescreen' contains deprecated bezel element\r\nWarning: layout view 'Hausjam_Artwork' contains deprecated bezel element\r\nWarning: layout view 'Upright_Artwork' contains deprecated bezel element\r\nWarning: layout view 'Upright_Artwork' contains deprecated backdrop element\r\nWarning: layout view 'XBLA_Widescreen' contains deprecated bezel element\r\nWarning: layout view 'Hausjam_Artwork' contains deprecated bezel element\r\n007766.20d.bin NOT FOUND (NO GOOD DUMP KNOWN) (tried in contra)\r\nWARNING: the machine might not run correctly.\r\n"
}
```
